src/components/Modal.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import ReactDom from 'react-dom';
+
+const modalStyles = {
```

Comment:
As you get ready to head into the design and styling portion of this project, you might want to decide as a group if you prefer to have `CSS in JS` such as it is here or if you prefer to have stylesheets which is what you all have been using.

I don't recommend having a combo of both as it can get very difficult to maintain as the code base grows.

Reply:
Agreed, we'll discuss this as team this week.